File: week_10/c_sharp_course/week_1/day_4/composition/checkpoint_1_composition/wizard_management_specs/RugTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class RugTest {
    Rug rug;

    [SetUp]
    public void Init() 
    {
      rug = new Rug("Yellow");
    }

    [Test]
    public void HasColour()
    {
      Assert.AreEqual("Yellow", rug.Colour);
    }
  }
}

File: week_06/day_3/composition/Composition_lab_solution/app/src/test/java/example/codeclan/com/composition_lab_solution/RugTest.java
package example.codeclan.com.composition_lab_solution;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_lab_solution.wizard_management.Rug;


public class RugTest {

    Rug rug;

    @Before
    public void before(){
        rug = new Rug("Yellow");
    }

    @Test
    public void hasColour(){
        assertEquals("Yellow",rug.getColour());
    }

}
